docs/updating/from_4.0/to_4.1.md

```diff
@@ -5,89 +5,161 @@ latest_tag: '4.1.5'
 
 # Update from v4.0.x to v4.1
 
-This update procedure applies if you are using v4.0.latest.
+This update procedure applies if you are using v4.0.0.
```

Comment:
Is this statement true? What about users who have a version that is in between 4.0.0 and 4.0.(latest-1)

Maybe, just maybe, we should split the procedure 3.3-like into "Update from v4.0.x to v4.0.latest" and "Update to 4.1"?
and begin the two articles with:
"This update procedure applies if you are using a v4.0 installation without the latest maintenance release."
and
"This update procedure applies if you are using v4.0.latest."
respectively?

Reply:
Th catch here is - 4.0 is not supported anymore, so we don't want users to update to it unless it's a stepping stone to v4.1+